include/meta/meta.hpp

```diff
@@ -2837,6 +2841,63 @@ namespace meta
     /// \endcond
 #endif
 
+#if META_CXX_VISIT
+    ///////////////////////////////////////////////////////////////////////////////////////////
+    // visit
+    /// \cond
+    namespace detail
+    {
+        
+        struct visit_fn
+        {
+            template<typename Fn, typename Type>
+            struct visit{
+                static constexpr void call(Fn f){ f(Type{}); }
```

Comment:
I had to use this struct dispatch to defer no to have `void` as Type instantiate
in `FnPtr`.